playground/backend/internal/code_processing/code_processing.go

```diff
@@ -336,12 +336,13 @@ func cancelCheck(ctx context.Context, pipelineId uuid.UUID, cancelChannel chan b
 		case <-ticker.C:
 			cancel, err := cacheService.GetValue(ctx, pipelineId, cache.Canceled)
 			if err != nil {
-				continue
+				logger.Errorf("%s: Process: cache.GetValue: error: %s", cancel, err.Error())
 			}
 			if cancel.(bool) {
 				cancelChannel <- true
+				return
 			}
-			return
+			continue
```

Comment:
it seems that there is no need to use `continue` here.

Reply:
Done.